gini/gini-sdk-ios
# Add accounting compatibility

Added accounting API compatibility through the SDK builder.

### How to test
Build and SDK and use the `GINIAPIAccountingType`, check the requests and see that they point to a different API.

### Merging
Automatic
